### DISCUSSION TITLE
关于使用Fooocus  api的一些事情

### DISCUSSION THREAD

**Discussion by feeddaidai** [mrhan1993/Fooocus-API#365] [General]
通过和stable diffusion的各种对比，依然选择使用Fooocus作为个人一个应用项目的核心驱动，虽然sd拥有更多的人群和拓展，但是在api的生态方面却非常的欠缺，作为使用AIGC作为微服务的架构使用，丰富的api内容才是首选。
经过对比，sd在同样运行迭代步数30的任务下，处理速度是Fooocus  api的一半，但是Fooocus api自带的异步以及步骤预览是非常好的！我猜想是这些阶段增加了一些耗时，不过结果还是很理想的。
目前很难解决的就是并发 和并行的解决方案，在Fooocus里，模型和lora的不停切换非常流畅，而sd的lora是在prompt中定义，模型切换却显得比较臃肿，目前我也没看到关于stable diffusion的并发并行解决方案，我依然想在Fooocus api中完成这个工作。

**Comment by mrhan1993** [comment]
并行的问题Fooocus不支持，我这里也是搞不了的==
